src/components/todo/Todo.js

```diff
@@ -13,8 +22,33 @@ class Todo extends Component {
   }
 
   render() {
-    return <div>화이팅^^</div>;
+
+    return (
+      <Card>
+        <TodoInput onSubmit={
+          function (content) {
+            if(content==''){ //아무 것도 입력하지 않았을 때
+              alert('TODO를 입력하고 ADD 하세요!');
+            }
+            else{ //입력값이 들어왔을 때
+              let list = [...this.state.todoList, content]; //기존 배열에 입력값 추가한 새 배열 생성
+              this.setState({
+                todoList: list
+              }); //react에게 state의 변경사항 알림
+            }
+          }.bind(this)
+        }></TodoInput>
+        <TodoCard data={this.state.todoList}
+          onClicked={function (i) {
+            let list = [...this.state.todoList]; //기본 배열과 값이 같은 새 배열 생성
+            list.splice(i, 1); //클릭한 내용만 삭제
+            this.setState({
+              todoList: list
```

Comment:
```suggestion
              todoList: [...this.state.todoList.slice(0, index), ...this.state.todoList.slice(index + 1, this.state.todoList.length)]
```
라고 바꿀 수 있습니다.
```suggestion
              todoList: [todoList.slice(0, index), ...todoList.slice(index + 1, todoList.length)]
```
위에서 const { todoList } = this.state; 라고 선언해줬다면 이렇게 훨씬 깔끔해지구요!
이를 destructure assignment라고 합니다. [참고할만한 글](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)에서 객체 구조 분해 부분을 보시면 이해되실거에요!